R/assertions.R

```diff
@@ -1557,20 +1560,29 @@ assert_list_element <- function(list, element, condition, message_text, ...) {
   )
   if (any(invalids)) {
     invalids_idx <- which(invalids)
-    abort(
-      paste0(
+
+    # construct supplementary message listing elements that are not correct type
+    if (is.null(message)) {
+      info_msg <- glue_collapse(
+        glue(
+          "{{.code {arg_name}[[{invalids_idx}]]${element} =
+           {lapply(list[invalids_idx], `[[`, element)}}}"
```

Comment:
Maybe not doable.  This is a bit hard on the eyes, but I guess if you are using `source_datasets` and `derive_var_extreme_dtm` then this wouldn't hurt you

![image](https://github.com/pharmaverse/admiraldev/assets/10111024/7a17b2b0-73b3-4073-96d3-cbc0fd42d0d2)

Reply:
Can you make a separate issue to discuss changing the messaging? We can deal with this in the second round, and here can focus on migrating to cli messaging? I 100% it's not easy to read.

Reply:
issue made - lets keep moving!